src/GovUk.Education.ExploreEducationStatistics.Content.Api/Services/ReleaseFileService.cs

```diff
@@ -85,14 +169,9 @@ private async Task<FileStreamResult> DownloadToStream(BlobInfo blob)
 
         private async Task<Either<ActionResult, BlobInfo>> GetBlob(string path)
         {
-            if (!await _blobStorageService.CheckBlobExists(PublicReleaseFiles, path))
-            {
-                return new NotFoundResult();
-            }
-
-            var blob = await _blobStorageService.GetBlob(PublicReleaseFiles, path);
+            var blob = await _blobStorageService.FindBlob(PublicReleaseFiles, path);
 
-            if (!blob.IsReleased())
```

Comment:
Cool, does this mean we can get rid of `IsReleased` and clean up Publisher to no longer be setting the release date time blob property, or still more cleanup to do?

Reply:
Not quite. Currently `MethodologyImageService` still uses this. Think we'd need to sort out the securing of that endpoint first to use the content DB.

I've created an overall ticket for removing the `IsReleased` meta from the blob anyway: https://dfedigital.atlassian.net/browse/EES-2619

Reply:
I forgot about the `MethodologyImageService`! Great, thanks for raising the ticket.